apps/app/src/views/components/TreatmentOptionSearch.tsx

```diff
@@ -0,0 +1,123 @@
+import { Box, ChakraProvider, InputGroup, InputRightElement, Spinner } from '@chakra-ui/react';
```

Comment:
first, here's a component that has a searchable dropdown backed by the treatmentOptions search (powered by medidata) that matches the UI elsewhere